lineage/figures/figure9.py

```diff
@@ -9,7 +9,18 @@
 from .figureCommon import getSetup, subplotLabel
 from ..tHMM import tHMM
 
-desired_num_states = np.arange(1, 5)
+desired_num_states = np.arange(1, 8)
+Ts = []
+PIs = []
+# to find the T and pi matrices to be used as the constant and reduce the number of estimations.
+for i in desired_num_states:
+    tHMM_solver = tHMM(X=Gemcitabine_Control, num_states=i)
+    tHMM_solver.fit()
+    # choose the estimated shape parameters for 1-state model to be kept constant
+    if i == 1:
+        constant_shape = [int(tHMM_solver.estimate.E[0].params[2]), int(tHMM_solver.estimate.E[0].params[4])]
+    Ts.append(tHMM_solver.estimate.T)
+    PIs.append(tHMM_solver.estimate.pi)
 
 
```

Comment:
tHMM uses whatever emissions are set within the data. So, here, you want to set them by `Gemcitabine_Control[i].E[j].G1.const_shape = const_G1`. You'll have to loop over every condition, lineage, state, and phase. It's weird to me that this is how the emissions are set, but it is what it is...

Reply:
I'm on it.